### PR TITLE
FCT-1187 - Filter Selection tests

### PR DESCRIPTION
**Pull request by valoriecarli**
added a few more unit tests and continued to expand on Filter int tests.

### CODE REVIEW

File: packages/components/filters/src/filters.spec.utils.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+//Click `Add filter` button & return dialog for interaction
+export const openAddFilterDialog = async (): Promise<HTMLElement> => {
+  const addFilterButton = await getAddFilterButton();
+  fireEvent.click(addFilterButton);
+  return await screen.findByRole('dialog');
+};
+
+//Select filter to apply
+export const selectFilter = async (dialog: HTMLElement, optionText: string) => {
```
Comment (ByronDWall) [inline packages/components/filters/src/filters.spec.utils.tsx:41]:
```suggestion
export const selectFilterOption = async (dialog: HTMLElement, optionText: string) => {
```

File: packages/components/filters/src/filters.spec.utils.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  });
+};
+
+export const getClearAllFiltersButton = (): HTMLElement | null => {
+  return screen.queryByRole('button', {
+    name: /clear all/i,
+  });
+};
+
+export const getBadgeStatus = (): HTMLElement | null => {
```
Comment (ByronDWall) [inline packages/components/filters/src/filters.spec.utils.tsx:22]:
```suggestion
export const getBadgeElement = (): HTMLElement | null => {
```

File: packages/components/filters/src/filters.spec.tsx [hunk 3]
```diff
@@ -57,73 +82,207 @@ const FilterTestComponent = (
...
+    const chips = displayedChips('primaryColors');
+    expect(chips).toEqual(['blue', 'green', 'pink', 'lavender', 'azure']);
+
+    //Collapse filter list & expect no badge to display
+    await toggleFilterList();
+    const filterTotalBadge = getBadgeStatus();
+    expect(filterTotalBadge).toBeNull();
+  });
+
+  //!is this needed?
```
Comment (ByronDWall) [inline packages/components/filters/src/filters.spec.tsx:190]: yes, it's good to have

**Review by tylermorrisford — APPROVED** [review]
I don't have the level of context that others have in the realm of filters, but this looks real nice 🪄

**Review by ddouglasz — APPROVED** [review]
Looks good! 👏

**Review by ByronDWall — APPROVED** [review]

**Pull request merged**